invenio_jobs/assets/semantic-ui/js/invenio_jobs/administration/JobActions.js

```diff
@@ -61,50 +68,41 @@ export class JobActions extends Component {
   };
 
   handleSuccess = () => {
+    const {resource} = this.props;
     this.setState({
       modalOpen: false,
       modalHeader: undefined,
       modalBody: undefined,
     });
     setTimeout(() => {
-      window.location.reload();
+      window.location = resource.links.self_admin_html;
```

Comment:
What is this timeout of 1 sec?

Reply:
This is when we create a new job run and we switch to detail view of this run (to monitor it).
@0einstein0 was the 1s chosen for any specific reason?